deploy/metrics.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: allocation-api
+  namespace: offender-management-staging
+  labels:
+    k8s-apps: allocation-api
+spec:
+  jobLabel: allocation-api
+  selector:
+    matchExpressions:
+    - {key: allocation-api, operator: Exists}
+  namespaceSelector:
+    matchNames:
+    - offender-management-staging
+    - monitoring
```

Comment:
Your `Service` will be in the `offender-management-staging` namespace so you don't need `monitoring` here.

Reply:
👌